### PR TITLE
Fixed telekinesis putting beaker in hand when swapped

### LINKED ISSUE #24684: Beaker swapping with Telekinesis
### Exploit Reports

- [X] I confirm this issue is not an exploit. (Required)

### BYOND Version

515.1620

### Issue Description

I tried to put a glass into full soda fountain, the glass that was already inside the fountain teleported into my hand

### What did you expect to happen?

I expected for glass inside to drop to the ground near the soda fountain

### What happened instead?

I now have coffee inside my hand

### Why is this bad/What are the consequences?

This was unintended effect of my beaker swapping PR #22915
```dm
if(beaker)
    user.put_in_hands(beaker)
```
You should not be able to teleport things into your hands from the distance

### Steps to reproduce the issue.

1. Have telekenesis
2. Find machine that can swap beakers, such as soda fountain or chem-dispenser
3. Load beaker into machine
4. Try to load another beaker into machine using telekinesis

### When did the problem start happening?

#22915

### PR DESCRIPTION
**Pull request by EmeraldCandy**
## What Does This PR Do
fixes #24684

Doesnt put a beaker in hand when swapping it out with telekinesis

## Why It's Good For The Game
bugs bad

## Testing
spawned in as chemist
stepped near a chem dispenser and placed in/swapped beakers
stepped away and couldnt insert beakers
gave telekinesis to self
inserted beaker into machine, beaker popped out on top of dispenser

## Changelog
:cl:
fix: Fixed telekinesis putting beaker in hand when swapped
/:cl:

### CODE REVIEW

**Review by Sirryan2002 — APPROVED** [review]
I recommend adding a comment about telekinesis next to your checks

**Comment by EmeraldCandy** [comment]
> I recommend adding a comment about telekinesis next to your checks

Done! :>

File: code/modules/reagents/chemistry/machinery/chem_dispenser.dm [hunk 1]
```diff
@@ -251,8 +251,11 @@
 			return
 		I.forceMove(src)
 		if(beaker)
-			user.put_in_hands(beaker)
 			to_chat(user, "<span class='notice'>You swap [I] with [beaker].</span>")
+			if(Adjacent(usr) && !issilicon(usr)) //Prevents telekinesis from putting in hand
```
Comment (DGamerL) [inline code/modules/reagents/chemistry/machinery/chem_dispenser.dm:255]: Use `user` not `usr` here

File: code/modules/reagents/chemistry/machinery/chem_master.dm [hunk 2]
```diff
@@ -125,8 +125,11 @@
 			return
 		I.forceMove(src)
 		if(beaker)
-			user.put_in_hands(beaker)
 			to_chat(user, "<span class='notice'>You swap [I] with [beaker] inside.</span>")
+			if(Adjacent(usr) && !issilicon(usr)) //Prevents telekinesis from putting in hand
```
Comment (DGamerL) [inline code/modules/reagents/chemistry/machinery/chem_master.dm:129]: Same here

**Review by DGamerL — CHANGES_REQUESTED** [review]

**Review by DGamerL — APPROVED** [review]

**Pull request merged**